lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -426,6 +426,16 @@ public static function singleIdNotAllowedOnCompositePrimaryKey($entity)
 
     /**
      * @param string $entity
+     *
+     * @return MappingException
+     */
+    public static function noIdDefined($entity)
+    {
+        return new self('No ID defined for entity '.$entity);
```

Comment:
Please add spaces before and after the concatenation operator (`'No ID defined for entity ' . $entity`)